jenkins-design-language/src/js/components/material-ui/svg-icons/action/accessibility.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import pure from 'recompose/pure';
```

Comment:
No need to change the names I think, unless it really bothers Cliff :) But we could probably just write a script to copy them and strip the recompose.pure stuff mechanically. I'm happy to help with that if you like, since I'm the one suggesting it.